sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/webpack-assets/build.sbt

```diff
@@ -51,10 +47,14 @@ InputKey[Unit]("html") := {
   assert(files.length == assetsCount)
   // Check all files are present
   assert(files.map(_.data.exists).forall(_ == true))
+  // There is only one app file
+  assert(files.count(_.metadata.get(BundlerFileTypeAttr) == Some(BundlerFileType.Application)) == 1)
   // There is only one library file
   assert(files.count(_.metadata.get(BundlerFileTypeAttr) == Some(BundlerFileType.Library)) == 1)
-  // And 2 assets, the css and its map
-  assert(files.count(_.metadata.get(BundlerFileTypeAttr) == Some(BundlerFileType.Asset)) == 2)
+  // There is only one library file
+  assert(files.count(_.metadata.get(BundlerFileTypeAttr) == Some(BundlerFileType.Loader)) == 1)
+  // And 1 asset (HTML file)
+  assert(files.count(_.metadata.get(BundlerFileTypeAttr) == Some(BundlerFileType.Asset)) == 1)
```

Comment:
This is odd, I'd expect to see the css as an asset

Reply:
It was near 2 months ago, but if I remember correctly the stats for these files has been moved to `stats.relatedAssets` (https://webpack.js.org/blog/2020-10-10-webpack-5-release/)